### PR TITLE
NAS-101078

### PR DESCRIPTION
**Pull request by dkmullen**
NAS-101078  for stable branch
Adds a pull-down on SMB form that populates with group list

### CODE REVIEW

File: src/app/helptext/services/components/service-smb.ts [hunk 1]
```diff
@@ -68,6 +68,13 @@ cifs_srv_guest_tooltip: T('Account to be used for guest access. Default is\
  the shared pool or dataset.\
  When the Guest Account user is deleted it resets to nobody.'),
 
+cifs_srv_admin_group_placeholder: T('Administrators Group'),
+cifs_srv_admin_group_tooltip: T('Members of this group are local admins\
```
Comment (Mrt134) [inline src/app/helptext/services/components/service-smb.ts]:
```suggestion
cifs_srv_admin_group_tooltip: T('Members of this group are local\
```

File: src/app/helptext/services/components/service-smb.ts [hunk 2]
```diff
@@ -68,6 +68,13 @@ cifs_srv_guest_tooltip: T('Account to be used for guest access. Default is\
...
+ and automatically have privileges to take ownership of any file in an SMB\
```
Comment (Mrt134) [inline src/app/helptext/services/components/service-smb.ts]:
```suggestion
 administrators that can take ownership of any file in an SMB\
```

File: src/app/helptext/services/components/service-smb.ts [hunk 3]
```diff
@@ -68,6 +68,13 @@ cifs_srv_guest_tooltip: T('Account to be used for guest access. Default is\
...
+ share, reset permissions, and administer the SMB server through the\
+ Computer Management MMC snap-in.'),
```
Comment (Mrt134) [inline src/app/helptext/services/components/service-smb.ts]:
```suggestion
 Microsoft Management Console (MMC).'),
```

File: src/app/helptext/services/components/service-smb.ts [hunk 4]
```diff
@@ -68,6 +68,13 @@ cifs_srv_guest_tooltip: T('Account to be used for guest access. Default is\
  the shared pool or dataset.\
  When the Guest Account user is deleted it resets to nobody.'),
 
+cifs_srv_admin_group_placeholder: T('Administrators Group'),
+cifs_srv_admin_group_tooltip: T('Members of this group are local admins\
+ and automatically have privileges to take ownership of any file in an SMB\
+ share, reset permissions, and administer the SMB server through the\
```
Comment (Mrt134) [inline src/app/helptext/services/components/service-smb.ts]:
```suggestion
 share, reset permissions, and administer the SMB server with the\
```

**Comment by dkmullen** [comment]
rebuild

**Comment by erinix** [comment]
rebuild

**Pull request merged**